.gitignore

```diff
@@ -9,3 +9,5 @@ __pycache__/
 # editor-generated files
 .vscode/
 .idea/
+
+*.pdf
```

Comment:
Could you also add other image types to gitignore? say img and stuff like that.
```
# Images and Documents
*.jpg
*.jpeg
*.png
*.pdf

Reply:
sure, thanks

Reply:
Looks reasonable. :)